elasticsearch/class-elasticsearch.php

```diff
@@ -34,6 +34,11 @@ protected function setup_constants() {
 
 	protected function setup_hooks() {
 		add_filter( 'ep_index_name', [ $this, 'filter__ep_index_name' ], PHP_INT_MAX, 3 ); // We want to enforce the naming, so run this really late.
+
+		// Network layer replacement to use VIP helpers (that handle slow/down upstream server)
+		add_filter( 'ep_intercept_remote_request', '__return_true', 9999 );
+		add_filter( 'ep_do_intercept_request', [ $this, 'filter__ep_do_intercept_request' ], 9999, 4 );
+		add_filter( 'jetpack_active_modules', [ $this, 'filter__jetpack_active_modules' ], 9999 );
```

Comment:
I'd do `PHP_INT_MAX` here just to be absolutely sure we disable Jetpack's module

Reply:
I think `9999` is better for this one, b/c it allows overridding if really needed. For example, maybe JP Search provided a certain piece of functionality that VIP ES doesn't and a client wanted to selectively use that part (disabling other parts of JP Search).